src/status_im2/contexts/activity_center/view.cljs

```diff
@@ -214,15 +214,16 @@
     (rf/dispatch [:activity-center.notifications/fetch-first-page])
     (fn []
       (let [notifications (rf/sub [:activity-center/notifications])]
-        [rn/view {:flex 1 :padding-top (navigation/status-bar-height)}
-         [blur/view style/blur]
-         [header]
-         [rn/flat-list
-          {:data                      notifications
-           :render-data               active-swipeable
-           :content-container-style   {:flex-grow 1}
-           :empty-component           [empty-tab]
-           :key-fn                    :id
-           :on-scroll-to-index-failed identity
-           :on-end-reached            #(rf/dispatch [:activity-center.notifications/fetch-next-page])
-           :render-fn                 notification-component}]]))))
+        [theme/provider :dark
```

Comment:
Quick Question: Since the app root is wrapped with the theme provider. Is there a way we can pass the theme key (`:light` or `:dark`) as screen params just like `:sheet?` and send it to the provider? If yes, that would reduce the wrapping of the screen component again with the specified theme key.

Reply:
yeah that sounds like a nice solution. Seems like it would work to me We can definitely try this approach out!  :)